Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ExternalCalls.aspx.cs

```diff
@@ -20,6 +20,11 @@ public partial class ExternalCalls : System.Web.UI.Page
         private const string ConnectionString = @"Data Source=.\SQLEXPRESS;Initial Catalog=RDDTestDatabase;Integrated Security=True";
 
         /// <summary>
+        /// Invalid connection string to database.
+        /// </summary> 
+        private const string InvalidConnectionString = @"Data Source=invalid\SQLEXPRESS;Initial Catalog=RDDTestDatabase;Integrated Security=True";
```

Comment:
Invalid account credentials is good - we connect to sql and fail to open. (the original one was failing to even find a sql instance..hence i suggested adding more scenario)